.env

```diff
@@ -1,2 +1,3 @@
+REACT_APP_WEBMAP_DOMAIN=https://map.treetracker.org
```

Comment:
Looks like you've created this branch from another dev branch rather than `master` and therefore have carried across some commits that aren't related to this fix.

Reply:
Oh that was why I had so many git errors, thank you!